<launch>
  <push-ros-namespace namespace="traffic_light_selector"/>
  <node_container pkg="rclcpp_components" exec="component_container" name="container" namespace="">
    <composable_node pkg="traffic_light_selector" plugin="TrafficLightSelector" name="node">
      <remap from="~/sub/vector_map" to="/map/vector_map"/>
      <remap from="~/sub/traffic_lights" to="/perception/traffic_light_recognition/traffic_signals"/>
    </composable_node>
  </node_container>
</launch>
